examples/s3-import/s3.tf

```diff
@@ -1,61 +1,113 @@
-resource "aws_s3_bucket" "mysql" {
-  #checkov:skip=CKV2_AWS_6: Ensure that S3 bucket has a Public Access block
-  #checkov:skip=CKV_AWS_19: Ensure all data stored in the S3 bucket is securely encrypted at rest
-  #checkov:skip=CKV_AWS_144: Ensure that S3 bucket has cross-region replication enabled
-  #checkov:skip=CKV_AWS_18: Ensure the S3 bucket has access logging enabled
-  #checkov:skip=CKV_AWS_145: "Ensure that S3 buckets are encrypted with KMS by default"
-  bucket        = local.name
-  force_destroy = true
-
+provider "aws" {
+  region = "eu-west-1"
```

Comment:
Can this be a variable or local?

Reply:
@hugoalmeida264  I have put this in provider.tf. Providers block not supported in locals, only the values